gpu-aware-scheduling/deploy/gas-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gpu-aware-scheduling
+  namespace: default 
+  labels:
+    app: gas
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gas
+  template:
+    metadata:
+      labels:
+        app: gas
+    spec:
+      serviceAccountName: gpu-aware-scheduling-service-account
+      containers:
+      - name: gasext
+        command:
+        - /extender
+        - --cert=/gas/cert/tls.crt
+        - --key=/gas/cert/tls.key
+        - --cacert=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        - -v
+        - "4"
+        image: registry.local/gpu-extender:1.0.0
```

Comment:
should this image be made to use the simple image name? Does registry.local resolve on all systems / CRIs?

Reply:
this is my mistake this yaml needs to be cleaned. intel normal location will be here